onnxruntime/test/providers/cpu/tensor/split_op_test.cc

```diff
@@ -431,7 +431,7 @@ TEST(SplitOperatorTest, InvalidValueInSplitAttribute) {
                               5.f, 6.f,
                               7.f, 8.f}};
 
-  std::vector<int64_t> splits{1, 0, 3};  // 0 is not valid
+  std::vector<int64_t> splits{1, -1, 3};  // 0 is not valid
   outputs.push_back({{1, 2}, {1.f, 2.f}});
```

Comment:
Comment needs updating.

Please add a unit test for your change as well to validate that a length of 0 is handled correctly.

Reply:
Thanks!